contributors/devel/sig-testing/verify-tests.md

```diff
@@ -0,0 +1,78 @@
+# Verification Tests
+
+**Table of Contents**
+
+- [Verification Tests](#verification-tests)
+  - [Overview](#overview)
+  - [`verify-govet-leveee`](#verify-govet-levee)
+
+## Overview
+
+Verification tests for Kubernetes provide a mechanism to verify contributions for adherence to project conventions
+and best practices, and to validate generated build artifacts for soundness.
+
+All blocking verification tests can be executed via `make verify`.
+Individual verification tests also can be found in vestigial shell scripts at `hack/verify-*.sh`.
+
+Most verification tests are self-explanatory.
+`verify-golint` and `verify-gofmt`, for instance, fail when a contribution does not adhere to lint and formatting conventions.
```

Comment:
The `golint` check was removed in https://github.com/kubernetes/kubernetes/pull/98063

Maybe `verify-govet` would be a good replacement here?

Reply:
Good catch.  `govet` is probably the good choice here, since `go-vet-levee` hooks directly into it.  It's unlikely that we'll miss that getting removed.